script.js

```diff
@@ -1 +1,252 @@
-console.log('yay!');
+// ======Global variables=====================
+let outputBox = null;
+let firstCardElement = null;
+let canClick = true;
+const firstCardColRowIndex = [];
+const gameDurationis3Min = 1000 * 60 * 3;
```

Comment:
This var might be more clearly expressed as `gameDuration`, with an inline comment that explains that it's 3 minutes.